infrastructure/critical/run_terraform.sh

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env bash
 
-AWS_CLI_PROFILE=${AWS_PROFILE:-platform-dev}
+AWS_CLI_PROFILE="catalogue-pipeline-terraform"
```

Comment:
Where is this profile from? Am I expected to have it in my `.aws/credentials`?

Reply:
This change creates that profile for you. Currently this works by expecting you to have a profile of a particular name, which is annoying because it expects everyone to set this up themselves and follow an implicit naming scheme. 

This uses the AWS CLI to configure that profile for you so you don't have to know or care.

Reply:
> Does this set it forever in was locally?

It'll try and recreate it every time you run it, it will overwrite a profile of the same name if it already exists. Overwriting is only a problem if someone has already created a profile of this name and is using it for something else, which I think is unlikely.

Reply:
I'm not sure if I am being silly - but where is it creating the profile?